website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -32,24 +32,19 @@ When users are provisioned, the following attributes are supported
 - Given name
 
 The following IdPs are supported in the <Constant name="cloud" /> UI:
-- Okta
-- Entra ID (_coming soon_)
+- [Okta](#scim-configuration-for-okta)
+- [Entra ID](#scim-configuration-for-entra-id)
 
 If your IdP isn’t on the list, it can be supported using <Constant name="cloud" /> APIs (_docs coming soon_).
```

Comment:
```suggestion
If your IdP isn’t on the list, it can be supported using <Constant name="cloud" /> [APIs](https://docs.getdbt.com/dbt-cloud/api-v3#/operations/Retrieve%20SCIM%20configuration).
```